### PR TITLE
Add hitTestBehavior and rootOverlay to LongPressDraggable constructor…

### LINKED ISSUE #78443: LongPressDraggable should have a flag to use rootOverlay and hitTestBehavior
## Use case

In scenarios that we use [LongPressDraggable](https://api.flutter.dev/flutter/widgets/LongPressDraggable-class.html), we might be needing to have an access to rootOverlay and hitTestBehavior. Even though, the parent class [Draggable](https://api.flutter.dev/flutter/widgets/Draggable-class.html) have these values at the current master channel, there is no access to either rootOverlay or hitTestBehavior.

## Proposal

Add two new variables to LongPressDraggable constructor and pass them with the `super` call which will take these two variables directly to the parent class.

I will create a Pull Request to fix this problem.

### PR DESCRIPTION
**Pull request by Amir-P**
- Fixes #78443

## Pre-launch Checklist

- [X] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [X] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [X] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [X] I signed the [CLA].
- [X] I listed at least one issue that this PR fixes in the description above.
- [X] I updated/added relevant documentation (doc comments with `///`).
- [X] I added new tests to check the change I am making, or this PR is [test-exempt].
- [X] I followed the [breaking change policy] and added [Data Driven Fixes] where supported.
- [X] All existing and new tests are passing.

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[Data Driven Fixes]: https://github.com/flutter/flutter/wiki/Data-driven-Fixes

### CODE REVIEW

**Review by goderbauer — APPROVED** [review]
LGTM

File: packages/flutter/test/widgets/draggable_test.dart [hunk 1]
```diff
@@ -3104,6 +3104,92 @@ void main() {
       );
     });
 
+  testWidgets('Drag feedback is put on root overlay with [rootOverlay] flag', (WidgetTester tester) async {
```
Comment (justinmc) [inline packages/flutter/test/widgets/draggable_test.dart:3107]: Nit: I think this isn't indented far enough, or the contents of the test are indented too far.

**Review by justinmc — APPROVED** [review]
LGTM with a nit, thanks for the PR!

File: packages/flutter/test/widgets/draggable_test.dart (hunk 1 above)
Comment (Amir-P) [reply packages/flutter/test/widgets/draggable_test.dart:3107]: I think it should be fixed now. @justinmc

**Review by justinmc — APPROVED** [review]
Indentation looks good now, thanks. Renewing my LGTM :+1:

**Pull request merged**